### PR TITLE
cam_cesm2_1_rel_48: Extend solar forcings to be able to complete extended SSP simulations…

### LINKED ISSUE #156: Need to update solar forcing file
It turns out the CMIP6 solar file was 2 days short for what the model needs to complete the SSP extension runs out to 2300. So for all future releases and development tags, please replace this file:

SolarForcingCMIP6_18491230-22991231_c171031.nc

with this file:

SolarForcingCMIP6_18491230-23000102_c20200615.nc

This can be done for all compsets that use the file.

### LINKED ISSUE #235: waccmx heat fluxes at equator
Here is a description of the change:

In CESM v2.1.3 and earlier, the electron heat flux was defined as following (in ionosphere.F90 ion_electron_temp.F90):

FeD = FeDCoef1 * f107 * FeDB - FeDCoef2 * f107

This was adapted from TIE-GCM, and has been determined to be incorrect by producing non-zero downward heat flux at the magnetic equator. The correct form of the heat flux is:

FeD = FeDCoef1 * f107 * FeDB

The value of FeDCoef1 has also been changed:

old: real(r8), parameter :: FeDCoef1 = -5.0E+7_r8

new: real(r8), parameter :: FeDCoef1 = -9.0E+7_r8

This change of value was suggested by Dr. Wenbin Wang.

Hanli

### PR DESCRIPTION
**Pull request by fvitt**
Update solar forcing input file
Fix WACCMX heat flux bug

closes #156 
closes #235

        modified:   bld/namelist_files/use_cases/hist_cam6.xml
        modified:   bld/namelist_files/use_cases/hist_trop_strat_vbs_cam6.xml
        modified:   bld/namelist_files/use_cases/hist_trop_strat_vbsext_cam6.xml
        modified:   bld/namelist_files/use_cases/hist_trop_strat_vbsfire_cam6.xml
        modified:   bld/namelist_files/use_cases/ssp126_cam6.xml
        modified:   bld/namelist_files/use_cases/ssp245_cam6.xml
        modified:   bld/namelist_files/use_cases/ssp370_cam6.xml
        modified:   bld/namelist_files/use_cases/ssp585_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_ma_2000_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_ma_hist_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_sc_hist_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_tsmlt_hist_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_tsmlt_ssp126_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_tsmlt_ssp245_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_tsmlt_ssp370_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_tsmlt_ssp534_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_tsmlt_ssp534ext_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_tsmlt_ssp585_cam6.xml
        modified:   bld/namelist_files/use_cases/waccm_tsmlt_ssp585ext_cam6.xml
        modified:   doc/ChangeLog
        modified:   src/physics/waccmx/ion_electron_temp.F90

### CODE REVIEW

**Review by cacraigucar — APPROVED** [review]

**Review by joemci — APPROVED** [review]

File: src/physics/waccmx/ion_electron_temp.F90 [hunk 1]
```diff
@@ -1156,13 +1156,15 @@ subroutine update_teti(state, dSETendIn, dSETendOut, ztodt, istate, tE, tI, teTi
...
-        if (ABS(dlatm(iCol)) < 40.0_r8) FeDB = 0.5_r8 * &
-                                        (1._r8 + SIN(pi * (ABS(dlatm(iCol)) - 20.0_r8) /40.0_r8))
-
-        if (ABS(dlatm(iCol)) >= 40.0_r8) FeDB = 1._r8
-
-
-        FeD = FeDCoef1 * f107 * FeDB - FeDCoef2 * f107
+        if (ABS(dlatm(iCol)) < 10.0_r8) then
+           FeDB = 0._r8
+        else if (ABS(dlatm(iCol)) >= 10.0_r8 .and. ABS(dlatm(iCol)) < 40.0_r8) then
```
Comment (gold2718) [inline src/physics/waccmx/ion_electron_temp.F90:1161]: If you get to this line, isn't `ABS(dlatm(iCol)) >= 10.0_r8` always `.true.`? It seems logically equivalent to just:
```suggestion
        else if (ABS(dlatm(iCol)) < 40.0_r8) then
```

File: doc/ChangeLog [hunk 2]
```diff
@@ -1,3 +1,29 @@
+Extend solar forcings to be able to complete extended SSP simulations
```
Comment (gold2718) [inline doc/ChangeLog:1]: I assume the rest of the ChangeLog entry will be added.

**Review by gold2718 — APPROVED** [review]
Just a minor potential simplification and a ChangeLog reminder.

File: src/physics/waccmx/ion_electron_temp.F90 (hunk 1 above)
Comment (fvitt) [reply src/physics/waccmx/ion_electron_temp.F90:1161]: @gold2718, I believe you are correct.  However, I would like to keep this code identical to what is in cam_development.

**Review by nusbaume — APPROVED** [review]

**Review by brian-eaton — APPROVED** [review]

**Pull request merged**